modules/nlu/src/views/full/intents/editor.jsx

```diff
@@ -242,17 +263,44 @@ export default class IntentsEditor extends React.Component {
             </h1>
           </div>
         </div>
-        <SplitterLayout secondaryInitialSize={350} secondaryMinSize={200}>
-          {this.renderEditor()}
-          <div className={style.entitiesPanel}>
-            <Slots
-              ref={el => (this.slotsEditor = el)}
-              axios={this.props.axios}
-              slots={this.state.slots}
-              onSlotsChanged={this.handleSlotsChanged}
-            />
-          </div>
-        </SplitterLayout>
+        <div className={classnames('pull-left', style.selectContext)}>
+          <OverlayTrigger
+            placement={'right'}
+            overlay={
+              <Tooltip id={`tooltip-right`}>
+                You can type in the select bar to add new contexts. To learn more about contexts, try the Welcome Bot.
+              </Tooltip>
+            }
+          >
+            <i className="material-icons md-14">info</i>
+          </OverlayTrigger>
+          <label for="selectContext">Current contexts</label>
+          <Creatable
+            id="selectContext"
+            multi
+            onChange={this.handleChangeContext}
+            value={this.state.selectedContextOptions}
+            options={
+              this.state.availableContexts &&
+              this.state.availableContexts.map(x => {
+                return { value: x, label: x }
+              })
+            }
+          />
+        </div>
```

Comment:
What do you think of moving the dropdown menu to the right of the label instead of below? just so we can see more utterances. Also, the context seems a bit out of place (especially when scrolling down), maybe a line + spacing between it and the utterance would be nice ? 

![image](https://user-images.githubusercontent.com/42552874/55842875-64761300-5b03-11e9-8641-fe7eae998e1a.png)